### PR TITLE
[3456] modified advertising string, removed TierPrice rendering

### PR DESCRIPTION
**Pull request by pablobentivengo**
**Related issue(s):**
* Fixes https://github.com/scandipwa/scandipwa/issues/3456

**Problem:**
* Advertising message shows Price value per item.
* Product Card rendered "As Low As" tag price

**In this PR:**
* Modified string for advertising message
* Removed As Low As tag

### CODE REVIEW

**Comment by aleksandrakorolova** [comment]
1. Apply fixes mentioned in the task only for bundle product (display message in this format 'Buy 2 with 10% discount each')
2. Please, return previous logic for all products except bundle product (display message in this format 'Buy 2 for US$14.04 each and save 10%')
Screenshot 2022-02-02 at 12 51 16

3. Also, return 'As low as'  on PLP and PDP for all products except bundle product
Screenshot 2022-02-02 at 12 51 38

**Comment by aleksandrakorolova** [comment]
1. Please, add word '**discount**' for bundle product as I wrote in previous comment ('Buy 2 with 10% **discount** each')
2. Fixed
3. Fixed

File: packages/scandipwa/src/component/TierPrices/TierPrices.component.js [hunk 1]
```diff
@@ -45,31 +46,58 @@ export class TierPrices extends PureComponent {
...
+                formattedPrice
+            ) }
                 <strong>
                     { __(
                         'save %s%',
                         Math.round(percent_off)
                     ) }
                 </strong>
-            </li>
+            </>
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/TierPrices/TierPrices.component.js:83]: I assume indent out here is not correct

File: packages/scandipwa/src/component/TierPrices/TierPrices.component.js [hunk 2]
```diff
@@ -45,31 +46,58 @@ export class TierPrices extends PureComponent {
...
+                'Buy %s with ',
+                quantity
+            ) }
+            <strong>
+                { __(
+                    '%s% discount each',
+                    Math.round(percent_off)
+                ) }
+            </strong>
+            </>
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/TierPrices/TierPrices.component.js:100]: Same here

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]
Looks good for me.
Some minor changes required.

File: packages/scandipwa/src/component/TierPrices/TierPrices.component.js [hunk 3]
```diff
@@ -45,31 +46,58 @@ export class TierPrices extends PureComponent {
...
+                },
+                type_id
             }
         } = this.props;
 
         // Don't show offers that make no sense
         if (value >= minPriceForOneUnit) {
             return null;
         }
-
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/TierPrices/TierPrices.component.js:56]: Return white space.

Comment (pablobentivengo) [reply packages/scandipwa/src/component/TierPrices/TierPrices.component.js]: Done!

File: packages/scandipwa/src/component/TierPrices/TierPrices.component.js (hunk 1 above)
Comment (pablobentivengo) [reply packages/scandipwa/src/component/TierPrices/TierPrices.component.js]: Done!

File: packages/scandipwa/src/component/TierPrices/TierPrices.component.js (hunk 2 above)
Comment (pablobentivengo) [reply packages/scandipwa/src/component/TierPrices/TierPrices.component.js]: Done!

**Pull request merged**